common/generate_container_user

```diff
@@ -2,7 +2,8 @@
 USER_ID=$(id -u)
 GROUP_ID=$(id -g)
 
-if [ x"$USER_ID" != x"0" -a x"$USER_ID" != x"1001" ]; then
+grep -e '^[^:]*:[^:]*:'$USER_ID':' /etc/passwd
+if [ "$?" != 0 ]; then
```

Comment:
very well might, I'll try it.
That's the problem with bash, so many ways to do things. :)